docs/modules/getting_started/pages/basic_walkthrough.adoc

```diff
@@ -70,7 +70,7 @@ For information on displaying other data in a dashboard, see <<bw-panel-create,
 . In the left navigation menu, click *Dashboards*.
 . Click menu:New[New Dashboard].
 . Click *Add Visualization*.
-. Choose your data source (for example, *OpenNMS Entities*).
+. Choose your data source (for example, *OpenNMS Performance*).
```

Comment:
This must be Entities for an Alarm Table visualization.
```suggestion
. Choose your data source (for example, *OpenNMS Entities*).
```

Reply:
Thanks. This documentation was the first thing I wrote when I started at ONMS 4 years ago. I believe Jesse had suggested starting with Entities data source. We could create a ticket to change the walkthrough to be Performance data source focused if that makes more sense.